### PR TITLE
[bitnami/mongodb] Ensuring the .Values.auth.rootUser is used everywhere

### PR DESCRIPTION
**Pull request by dtrts**
### Description of the change

During the initial set up of a new node the setup.sh script attemts to connect to an existing node to verify if the cluster exists and to register as part of the existing replica set. 

This search had the root username hardcoded as `root`, however this is a value which can be altered by the user. 

### Benefits

This will ensure that a custom root username is honored and causes no unexpected issues when setting up new nodes in a relpicaset. 

### Possible drawbacks

None :) 

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for your suggestion, please note it is needed to only apply changes to one chart per PR, in this case, there are changes affecting two different charts. Could you please review that? Thanks

**Comment by dtrts** [comment]
@carrodher - You responded too quickly! 😄  

I have fixed the rebase issues and the PR is in a proper state.

**Review by aoterolorenzo — DISMISSED** [review]
LGTM!

@dtrts thank you for the contribution!

**Review by aoterolorenzo — APPROVED** [review]

File: bitnami/mongodb/templates/replicaset/scripts-configmap.yaml [hunk 1]
```diff
@@ -219,7 +219,7 @@ data:
     SLEEP_PERIOD=10
 
     {{- if and .Values.auth.enabled .Values.auth.rootPassword }}
-    usernameAndPassword="-u root -p ${MONGODB_ROOT_PASSWORD}"
+    usernameAndPassword="-u ${$MONGODB_ROOT_USER} -p ${MONGODB_ROOT_PASSWORD}"
```
Comment (slonopotamus) [inline bitnami/mongodb/templates/replicaset/scripts-configmap.yaml:222]: Shouldn't this be just `${MONGODB_ROOT_USER}`, without extra `$`?

Comment (dtrts) [reply bitnami/mongodb/templates/replicaset/scripts-configmap.yaml:222]: It absolutely should! I apologise for letting this slip through.

Comment (dtrts) [reply bitnami/mongodb/templates/replicaset/scripts-configmap.yaml:222]: I've implemented a fix here: https://github.com/bitnami/charts/pull/17116

**Pull request merged**